guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@tahb 👍 

What happens when you have a local version of the branch and then checkout the remote? Do you delete your local branch in order to replace it with the remote version?